src/App.tsx

```diff
@@ -16,18 +16,21 @@ import Pools from '@pages/Assets/Pools/Pools';
 import Tokens from '@pages/Assets/Tokens/Tokens';
 import Discover from '@pages/Discover/Discover';
 import MyAssets from '@pages/MyAssets/MyAssets';
+import NewNftMint from '@pages/MyAssets/NftMint/NewNftMint';
```

Comment:
Could it be named `NftMint` instead of `NewNftMint`?

Reply:
a, wait, I see the `NftMint` page already exists

Reply:
check https://github.com/SaltyCucumber/assets-portal/pull/55/files#diff-96453fc4941066bdda4cac7e2cc6e541a9d18657f1f03499f36dc6f47e3cd5bd

I've layed ground for the new flow, will remove/rename old one once that is done